development/mimir-with-otlp/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.4'
+services:
+
+  consul:
+    image: consul
+    command: [ "agent", "-dev" ,"-client=0.0.0.0", "-log-level=info" ]
+    ports:
+      - 8500:8500
+
+  minio:
+    image: minio/minio
+    command: [ "server", "/data" ]
+    environment:
+      - MINIO_ACCESS_KEY=mimir
+      - MINIO_SECRET_KEY=supersecret
+    ports:
+      - 9000:9000
+    volumes:
+      - .data-minio:/data:delegated
+
+  prometheus:
+    image: prom/prometheus:v2.36.2
+    command: ["--config.file=/etc/prometheus/prometheus.yaml"]
+    depends_on:
+      - mimir-1 
+    volumes:
+      - ./config:/etc/prometheus
+    ports:
+      - 9090:9090
+
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.54.0
+    command: ["--config=/etc/otel-collector/otel-collector.yaml"]
+    depends_on:
+      - mimir-1 
+    volumes:
+      - ./config:/etc/otel-collector
+    ports:
+      - 8083:8083
```

Comment:
Is this otel-collector the only things different in `mimir-with-oltp`?

I have concerns about having to maintain one more development setup which is probably the same as the one for s3 + otel-collector.

Maybe I'm missing something, then maybe we could have a README.md in this `development/mimir-with-oltp` that would clarify?

Reply:
I have the same concern. I don't understand the need of a dedicated dev env.

Reply:
I would just remove `development/mimir-with-otlp/` and add OT collector to `development/tsdb-blocks-storage-s3`.